playbook/app/pb_kits/playbook/pb_selectable_card/_selectable_card.jsx

```diff
@@ -55,7 +57,9 @@ const SelectableCard = ({
     { 'checked': checked,
       'disabled': disabled,
       'enabled': !disabled }),
-  globalProps(props), className)
+  dark && 'dark' || '',
```

Comment:
Is this line essentially `dark ? 'dark' : ""`..if so, let's use this one since it's a more common pattern.

Reply:
Exactly yes, Ill change it over